.github/ISSUE_TEMPLATE/architecture-feature-request.md

```diff
@@ -0,0 +1,134 @@
+---
+name: Architecture Feature Request
+about: Request for an architecture feature for a bussiness case
+title: "[Architecture Decision Feature] YOUR_ISSUE_TITLE_HERE"
+labels: 'architecture'
+
+---
+
+# Feature proposal for architecture decision
+
+This ADR template emphasizes creating a business case for a decision, including critera, candidates, and costs.
+
+|  | Value |  Description | 
+|:--| ----- |:--- |  
+| **Title** |  | A short present tense imperative phrase, less than 50 characters, like a git commit message. | 
+| **Status** |  |  One of proposed, accepted, rejected, deprecated, superseded, etc |
+| **Depencencies** | | Which applications, expert groups, organizations does this feature impact |
+
+
+## Top-level
+
+* Title
+* Status
+* Evaluation criteria
+* Candidates to consider
+* Research and analysis of each candidate
+  * Does/doesn't meet criteria and why
+  * Cost analysis
+  * SWOT analysis
+  * Opinions and feedback
+* Recommendation
+* Acceptance Criteria
+ 
+
+## Evaluation criteria 
+
+Summary: explain briefly what we seek to discover and why;
+
+# Candidates to consider (Low-level deep dive)
+
+Summary: explain briefly how we discovered candidates, and draw attention to any outliers. 
+
+List all candidates and related options; what are we evaluating as potential solutions?;  
+
+
+## Research and analysis of each candidate
+
+explain briefly the research methods, and draw attention to patterns, clusters, and outliers.
+
+### Does/doesn't meet criteria and why
+
+### Cost analysis
+
+* Examples
+
+    * Licensing, such as contract agreements and legal commitments
+
+    * Training, such as upskilling and change management
+
+    * Operating, such as support and maintenance
+
+    * Metering, such as bandwidth and CPU usage
+
+
+### SWOT analysis
+
+* Strengths
+
+* Weaknesses
+
+* Opportunites
+
+* Threats
+
+### Internal opinions and feedback
+
+
+* Examples
+
+    * By the team, ideally written by the actual person
+
+    * From other stakeholders
+
+    * Quality attributes a.k.a. cross-functional requirements 
+
+### External opinions and feedback
+
+* Who is providing the opinion?
+
+## What are other candidates you considered?
+
+## What are you creating? 
+
+    * Examples
+
+    * B2B or B2C
+
+    * external-facing or employee-only
+
+    * desktop or mobile
+
+    * pilot or production
+
+    * monolith or microservices
+
+## How did you evaluate the candidates?
+
+## Why did you choose the winner?
+
+## What is happening since then?
+
+    * Examples
+
+    * How is the winner performing?
+
+    * What % of real-world production user traffic is flowing through the winner?
+
+    * What kinds of integrations are involved, such as with continuous delivery pipelines, content management systems, analytics and metrics, etc.?
+
+    * Knowing what you know now, what would you advise people to do differently?

```

Comment:
```suggestion
<!--
Please note down how you validated the ADR since the change.

    * Examples
    * How is the winner performing?
    * What % of real-world production user traffic is flowing through the winner?
    * What kinds of integrations are involved, such as with continuous delivery pipelines, content management systems, analytics and metrics, etc.?
    * Knowing what you know now, what would you advise people to do differently?
-->
```